ansible/README.md

```diff
@@ -43,5 +43,20 @@ to enter the SSH password:
 The `--tags` option tells Ansible to only execute tasks with the specified tag.
 In the previous examples, only tasks with the `setup` tag would be executed.
 
-Provide the data requested by the playbook (eg Jenkins admin user name/password
-and SSH keys locations) and wait automatic setup finish.
+The ansible playbooks will prompt you to provide the following data:
+
+Master:
+- Jenkins admin user name and password: used to access the Jenkins UI with 
+administrative privileges.
+- Jenkins private and public SSH key file path: used to communicate between the 
+master and the slave nodes.
+
+Slave:
+- Jenkins private and public SSH key file path: used to communicate between the master 
+and the slave nodes.
+- Upload server user's private SSH key file path: used to upload builds to the configured server.
+- GitHub user's private SSH key file path: used to push commits to GitHub in periodic pipelines.
+- Path to file containing known keys for GitHub and upload server hosts: used to recognize 
+the server identities. 
+
+Provide the data requested by the playbooks and wait automatic setup finish.
```

Comment:
I've applied the changes requested by @olavph (see [outdated comment](https://github.com/open-power-host-os/infrastructure/pull/106#discussion_r135278310)) 
I'd like to point out that it is good, indeed, to describe what each key means, but I during the execution of the playbooks I was still confused as to which ones are automatically generated and which ones are user generated (therefore with a pre-existing path provided by the user).

For example, in the slave playbook, the first item (Jenkins private and public SSH key) is automatically generated by the playbook during setup, so the user can just press Enter and proceed, while the following 2 items (Upload server and GitHub private SSH keys) refers to a key manually generated by the user in the previous steps (assuming the user is following the setup from README in infrastructure main page) so in the prompt the user will have to overwrite the default value. However, this info on what action the user should take is not clear in this tutorial.

Reply:
@mopsfelder I do not recommend using the variable names, since the prompts that appear on execution show a phrase instead of them (i.e. `Enter Jenkins private SSH key file path [~/.ssh/jenkins_id_rsa]:`). This makes it a bit harder to associate the prompts with this documentation.

Reply:
@YasminBeatriz None of the SSH keys are automatically generated. They must be previously created by the user and may be anywhere in its system (although a standard path is suggested by default).